core/src/epicli/cli/epicli.py

```diff
@@ -202,6 +202,8 @@ def apply_parser(subparsers):
                             help='Password that will be used to encrypt build artifacts.')
     optional.add_argument('--profile-ansible-tasks', dest='profile_ansible_tasks', action="store_true",
                             help='Enable Ansible profile_tasks plugin for timing tasks. (developer/debug option)')
+    optional.add_argument('--ping-retries', dest='ping_retries', type=int, required=False, action='store', default=5,
```

Comment:
IMO it would be nice to have arguments in alphabetic order but this can be done next time.